server.go

```diff
@@ -103,9 +104,15 @@ const (
 	// the maximum number transactions of the minimum realistic size (~206
 	// bytes) in approximately one hour of blocks on the main network.
 	//
-	// Since each hash in the cache will occupy 32 bytes, this value will result
-	// in around 718KiB memory usage plus the overhead of the structure.
-	maxRecentlyConfirmedTxns = 23000
+	// recentlyConfirmedTxnsFPRate is the false positive rate to use for the
+	// APBF used to track recently confirmed transactions.  It is set to a rate
+	// of 1 per 1 million which supports up to ~11.5 transactions/s before a
+	// single false positive would be seen on average and thus allows for plenty
+	// of future growth.
+	//
+	// These values result in about 183 KiB memory usage including overhead.
+	maxRecentlyConfirmedTxns    = 23000
```

Comment:
Looks like `maxRecentlyConfirmedTxns` should be moved up, it is now disjointed from its comments block.

Reply:
I actually had put them together intentionally since they are intertwined.  I've updated the comments to make that more clear and treat them as a group.